src/autoIncrement.ts

```diff
@@ -54,26 +54,32 @@ export function AutoIncrementSimple(
 
 /** The Schema used for the trackers */
 const IDSchema = new mongoose.Schema({
-  f: String,
-  m: String,
-  c: Number
+  field: String,
```

Comment:
Why change that to the full name?
-> it was smaller so that it requires less bandwidth and lower storage footprint